src/config.rs

```diff
@@ -1513,14 +1513,22 @@ impl Config {
 
     #[cfg(windows)]
     pub fn installed_config<'a>() -> Option<Cow<'a, Path>> {
-        if let Some(mut path) = ::std::env::home_dir() {
-            path.push("alacritty");
-            path.set_extension("yml");
-            if path.exists() {
-                return Some(path.into());
-            }
+        #[allow(deprecated)]
+        let old = ::std::env::home_dir()
+            .map(|path| path.join("alacritty.yml"));
+        let new = dirs::config_dir()
+            .map(|path| path.join("alacritty\\alacritty.yml"));
+        // TODO: Remove old configuration location warning (Deprecated 03/12/2018)
+        if let Some(old) = old.as_ref().filter(|old| old.exists()) {
+            warn!(
+                "Found old configuration at: '{}'. The file should be moved to the new location: '{}'.", 
```

Comment:
The file is not necessarily an `old` configuration. It's just the old configuration location. Stating that clearly takes a few too many words imho, but just removing the `old` here should prevent confusion I think. Just something like `Found configuration file at: '{}'`.

Also in the change log it's noted that the old path "usually" was %USERPROFILE%. Would it be possible that the old and new path is the same, thus creating an warning the user can't resolve, telling him to move it from /x/y/z to /x/y/z?

Reply:
`%APPDATA%` is a child of `%USERPROFILE%`, so they should never point to the same location.